src/Row.ts

```diff
@@ -0,0 +1,47 @@
+import { WidgetProperties } from '@dojo/widget-core/interfaces';
+import WidgetBase from '@dojo/widget-core/WidgetBase';
+import { RegistryMixin, RegistryMixinProperties } from '@dojo/widget-core/mixins/Registry';
+import { v, w } from '@dojo/widget-core/d';
+import { HasColumns } from './interfaces';
+import { CellProperties } from './Cell';
+import { theme, ThemeableMixin } from '@dojo/widget-core/mixins/Themeable';
+
+import * as rowClasses from './styles/row.css';
+
+export interface RowProperties extends WidgetProperties, HasColumns, RegistryMixinProperties {
+	item: any;
```

Comment:
Good catch. I like the field stuff as well. I'm going to make a note to go through and see where I can refine that.